docs/guide/snaps-development-guide.md

```diff
@@ -317,12 +314,18 @@ In the future, MetaMask will create some way for users to more easily discover s
 
 ## Resources and Tools
 
-You can review the growing number of [example snaps](https://github.com/MetaMask/snaps-skunkworks/tree/main/packages/examples) maintained by MetaMask, and you may wish to check out the first functional key management snap ever made, [the Filecoin Snap, or `filsnap`](https://github.com/Chainsafe/filsnap).
```

Comment:
I think we should include the list of example snaps as well, otherwise why are we creating them 😅